modules/installation-initializing.adoc

```diff
@@ -471,14 +471,20 @@ additionalTrustBundle: |
 The value must be the contents of the certificate file that you used for your mirror registry. The certificate file can be an existing, trusted certificate authority, or the self-signed certificate that you generated for the mirror registry.
 
 ifdef::aws+restricted[]
-.. Define the subnets for the VPC to install the cluster in:
+.. Define the subnets for the VPC to install the cluster in, as in the following example:
 +
 [source,yaml]
 ----
 subnets:
-- subnet-1
-- subnet-2
-- subnet-3
+- id: subnet-<id1>
+  roles:
+  - type: IngressControllerLB
+  - type: ControlPlaneLB
+  - type: Bootstrap
+- id: subnet-<id2>
+  roles:
+  - type: ControlPlaneLB
+  - type: ClusterNode
```

Comment:
```yaml
platform:
  aws:
    vpc:
      subnets:
        - id: subnet-<id1>
        - id: subnet-<id2>
        - id: subnet-<id3>
```
I suggest we can keep it simple by giving an example of specifying subnets without roles here.

- It is same as before but following new field convention.
- Since there is no explanation of such subnet roles here, it's best not to make it complex. And roles are optional also.
- Side note: There is no `ControlPlaneLB` role btw :D You might see it somewhere because we are lazy to say/type `ControlPlaneExternal` and `ControlPlaneInternalLB` haha